### PR TITLE
[hotfix] Fix Ray Tune output v2 failures [no_early_kickoff]

### PR DESCRIPTION
**Pull request by krfricke**
## Checks

- [ ] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
    - [ ] I've added any new APIs to the API Reference. For example, if I added a 
           method in Tune, I've added it in `doc/source/tune/api/` under the 
           corresponding `.rst` file.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Review by Yard1 — APPROVED** [review]

File: python/ray/tune/tune.py [hunk 1]
```diff
@@ -28,10 +28,21 @@
...
+        get_air_verbosity,
+        _detect_reporter as _detect_air_reporter,
+    )
+except Exception:
+
+    def get_air_verbosity(*args, **kwargs):
+        return None
+
+    def _detect_air_reporter(*args, **kwargs):
+        return None
```
Comment (matthewdeng) [inline python/ray/tune/tune.py:43]: Why is this needed?

**Review by xwjiang2010 — APPROVED** [review]

File: python/ray/tune/tune.py (hunk 1 above)
Comment (xwjiang2010) [reply python/ray/tune/tune.py:43]: because OrderedDict is not available for 3.6 :(
